docs/guides/prefect-deploy.md

```diff
@@ -883,6 +890,10 @@ These deployments can be managed independently of one another, allowing you to d
     Prefect supports multiple deployment declarations within the `prefect.yaml` file. This method of declaring multiple deployments allows the configuration for all deployments to be version controlled and deployed with a single command.
 
     New deployment declarations can be added to the `prefect.yaml` file by adding a new entry to the `deployments` list. Each deployment declaration must have a unique `name` field which is used to select deployment declarations when using the `prefect deploy` command.
+    
+    !!! warning
+        When using a custom `prefect.yaml` file, remember that the value for deployment `entrypoints` must be relative to 
```

Comment:
can we clarify what custom means here for the user? a diff name or location?

Reply:
Both! I added a clarification.

Reply:
ty!